uiowa/uiowa
# Search Block - External URLs

Resolves #5875

# How to test #
- pull branch, sync site
- Add search block with following:
   - endpoint: https://search.lib.uiowa.edu/primo-explore/search
   - query parameter: query
   - query prepend: any,contains,
   - additional parameters: 
       &institution=01IOWA&vid=01IOWA&mode=basic&tab=default_tab&search_scope=default_scope
- Use the block. Search should take you to results thru InfoHawk. 
- Add extra words, special characters, etc to the additional parameters text area. Hitting search should strip the excess out and still return results. 
- Check default search block field values still work as expected.

<details>
<summary>Original description</summary>
<p>For my professional dev time today, I wanted to look into why the block didn't support external URLs.
Relates to upcoming work https://github.com/uiowa/uiowa/issues/5875</p>
</details>
